core/const.go

```diff
@@ -12,13 +12,31 @@
  * the License.
  *
  * @author: Tingyu Zeng, Dell
- * @version: 0.1.0
+ * @version: 0.2.2
  *******************************************************************************/
 package main
 
+// Global constants
 const (
 	CertificatesPath = "certificates/"
 	SecurityService  = "securityservice"
 	EdgeXService     = "edgex"
 	VaultToken       = "X-Vault-Token"
+
+	// Vault API endpoints: v1
+	vaultHealthAPI      = "/v1/sys/health"
+	vaultInitAPI        = "/v1/sys/init"
+	vaultUnsealAPI      = "/v1/sys/unseal"
+	vaultPolicyAPI      = "/v1/sys/policy/"
+	vaultTokenCreateAPI = "/v1/auth/token/create"
+	vaultTokenDeleteAPI = "/v1/auth/token/delete"
+
+	vaultDefaultPolicy = "default"
+	vaultTokenTTL      = "168h"
```

Comment:
it would be nice if this was a configuration.toml parameter as well, but 168 hours is a sensible default